pinot-core/src/main/java/org/apache/pinot/core/query/executor/ServerQueryExecutorV1Impl.java

```diff
@@ -125,8 +125,8 @@ public synchronized void init(PinotConfiguration config, InstanceDataManager ins
     _planMaker.init(config);
     _defaultTimeoutMs = config.getProperty(Server.TIMEOUT, Server.DEFAULT_QUERY_EXECUTOR_TIMEOUT_MS);
     _enablePrefetch = Boolean.parseBoolean(config.getProperty(ENABLE_PREFETCH));
-    LOGGER.info("Initialized query executor with defaultTimeoutMs: {}, enablePrefetch: {}", _defaultTimeoutMs,
-        _enablePrefetch);
+    LOGGER.info("Initialized query executor with defaultTimeoutMs: {}, enablePrefetch: {}",
+        _defaultTimeoutMs, _enablePrefetch);
```

Comment:
(format) Per the changes in this file, I believe your IDE is not properly setup with [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup?q=null#set-up-ide)

Reply:
I have the rules properly set in my IDE, so not sure why this keeps happening. Have enforced them for the files changed by this PR, but I'm afraid there's a lot of files in the repository that should be reviewed as well (I tried to, but don't want my PR to pass from 6 to +400 files changed).